apps/src/gamelab/GameLab.js

```diff
@@ -177,6 +216,34 @@ GameLab.prototype.init = function (config) {
   });
 };
 
+function quote(str) {
+  return '"' + str + '"';
+}
```

Comment:
I know this is common, but this exact function now exists in:

* applab/applab.js
* applab/ChartApi.js
* studio/paramLists.js
* now this instance, gamelab/GameLab.js

Let's throw this into utils.js and be done with it :grin: